### ISSUE TITLE
Poetry does not respect command line arguments and .env arguments for uvicorn

### ISSUE THREAD

**Issue by callebtc** [lnbits/lnbits-legend#834]
1) `server.py` ignores `.env` variables `HOST` and `PORT`
2) Command line arguments are not passed to `uvicorn`. 

Example of what doesn't work:
```sh
poetry run --host 1.2.3.4 --reload
```

Example of how I usually start LNbits: 

```sh
./venv/bin/uvicorn lnbits.__main__:app --host 0.0.0.0 --port 5000 --ssl-keyfile ./localhost+2-key.pem --ssl-certfile ./localhost+2.pem
```

**Comment by callebtc** [comment]
@dni suggested the following, which works
```sh
poetry run uvicorn lnbits.__main__:app --host 0.0.0.0 --port 5000 --ssl-keyfile ./localhost+2-key.pem --ssl-certfile ./localhost+2.pem
```

This also seems to fix Issue #833. Do we update the docs or do we fix how `poetry run lnbits` starts LNbits?

**Comment by dni** [comment]
good question, i would suggest both commands in the docs.
